ios/Capacitor/Capacitor/Plugins/CapacitorUrlRequest.swift

```diff
@@ -40,8 +40,11 @@ open class CapacitorUrlRequest: NSObject, URLSessionTaskDelegate {
             throw CapacitorUrlRequestError.serializationError("[ data ] argument for request with content-type [ multipart/form-data ] may only be a plain javascript object")
         }
 
+        let allowed = CharacterSet(charactersIn: "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789-._")
```

Comment:
```suggestion
        let allowed = CharacterSet(charactersIn: "-._*").union(.alphanumerics)
```
This can be simplified with an union to `.alphanumerics` instead of typing them all
Also added `*` to match Android behavior, which uses `URLEncoder.encode` and according to the docs the `*` should remain the same:
> The special characters ".", "-", "*", and "_" remain the same.
https://docs.oracle.com/javase/8/docs/api/java/net/URLEncoder.html#encode-java.lang.String-java.lang.String-

I've noticed that this is also encoding the keys, tested and web also encodes the keys, but Android doesn't encode the keys.
Not sure which one should be the correct behavior, but I think all the 3 platforms should behave the same way, so all of them should encode the keys or none should.

Reply:
> Not sure which one should be the correct behavior, but I think all the 3 platforms should behave the same way, so all of them should encode the keys or none should.

According to [MDN](https://developer.mozilla.org/en-US/docs/Web/HTTP/Reference/Methods/POST), keys being encoded is the correct behavior.  However, changing it on Android could be seen as breaking, unless we categorize non-encoded keys as a bug (which I do).

What do you think?